Update serverless installation docs (#8116)
DataDog/documentation: content/en/serverless/installation/ruby.md

      text: 'Installing Ruby Serverless Monitoring'
---

After you have [installed the AWS integration][1], follow the steps below to instrument your application to send metrics, logs, and traces to Datadog.

## Configuration

### Install the Datadog Lambda Library

The Datadog Lambda Library can be imported as a layer or a gem.

The minor version of the `datadog-lambda` gem always matches the layer version. E.g., datadog-lambda v0.5.0 matches the content of layer version 5.

#### Using the Layer

> [Configure the layers][2] for your Lambda function using the ARN in the following format.

Reviewer: ```suggestion
[Configure the layers][2] for your Lambda function using the ARN in the following format:
```

For example:

```
arn:aws:lambda:us-east-1:464622532012:layer:Datadog-Ruby2-7:5
```

The available `RUNTIME` options are `Ruby2-5` and `Ruby2-7`. For more information, see the [latest release][3].

#### Using the Gem

Add the following line to your Gemfile. See the [latest release][4].

```
gem 'datadog-lambda'
gem 'ddtrace'
```

> Keep in mind that `ddtrace` uses native extensions, which must be compiled for Amazon Linux before being packaged and uploaded to Lambda. For this reason we recommend using the layer.

Reviewer: ```suggestion
Keep in mind that `ddtrace` uses native extensions, which must be compiled for Amazon Linux before being packaged and uploaded to Lambda. For this reason, Datadog recommends using the layer.
```

### Subscribe the Datadog Forwarder to the Log Groups

> You need to subscribe the Datadog Forwarder Lambda function to each of your function’s log groups, in order to send metrics, traces and logs to Datadog.

Reviewer 2: ```suggestion
You need to subscribe the Datadog Forwarder Lambda function to each of your function’s log groups to send metrics, traces and logs to Datadog.
```

> You need to subscribe the Datadog Forwarder Lambda function to each of your function’s log groups, in order to send metrics, traces and logs to Datadog.

Reviewer: ```suggestion
You need to subscribe the Datadog Forwarder Lambda function to each of your function’s log groups o send metrics, traces, and logs to Datadog.
```

Reviewer 2: fixed

1. [Install the Datadog Forwarder if you haven't][5].
2. [Ensure the option DdFetchLambdaTags is enabled][6].
3. [Subscribe the Datadog Forwarder to your function's log groups][7].

> ## Quick Start

Reviewer 2: ```suggestion
## Explore Datadog Serverless Monitoring
```

> After you have configured your function following the steps above, check out the sample code below to get started, and view metrics, logs and traces on the [Serverless page][8].

Reviewer 2: ```suggestion
After you have configured your function following the steps above, you should be able to view metrics, logs and traces on the [Serverless page][8]. If you need to submit a custom metric, refer to the sample code below:
```

> After you have configured your function following the steps above, you can view the enhanced metrics, logs and traces on the [Serverless page][8]. If you would like to submit a custom metric or manually instrument a function, see the sample code below.

Reviewer: ```suggestion
After you have configured your function following the steps above, you can view the enhanced metrics, logs and traces on the [Serverless page][8]. If you would like to submit a custom metric or manually instrument a function, see the sample code below:
```

Reviewer 2: fixed
